projects/storefrontapp-e2e-cypress/cypress/helpers/update-password.ts

```diff
@@ -80,7 +71,7 @@ export function updatePasswordTest() {
   });
 
   it('should be able to go to Update Password Page', () => {
-    accessUpdatePasswordPage();
+    verifyPasswordPage();
```

Comment:
Remove this assertion because I believe it is redundant and the other my account test doesn't have it as well